src/content.js

```diff
@@ -278,14 +278,33 @@ function markMergeCommitsInList() {
 }
 
 function indentInput(el, size = 4) {
+	const fullFieldText = el.value;
+	const {selectionStart, selectionEnd} = el;
```

Comment:
since you are destructing here, why didn't you destruct `value` in the same line?
if you insist of renaming `value` to `fullFieldText`, I'd would leverage [this approach](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names) to do that.

```js
const { selectionStart, selectionEnd, value: fullFieldText } = el
```